src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {
+  useState,
+  useContext,
+  useRef,
+  useEffect,
+} from 'react';
+import { useClickAway } from 'react-use';
+import { TodosContext } from '../../context/TodosContext';
+import { Todo } from '../../types/Todo';
+
+export const NewTodo = () => {
+  const [value, setValue] = useState('');
+  const { todos, updateTodos } = useContext(TodosContext);
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  const ref = useRef(null);
+
+  useEffect(() => {
+    if (inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [inputRef]);
+
+  const reset = () => {
+    setValue('');
+  };
+
+  const addNewTodo = () => {
+    const newTodo: Todo = {
+      id: todos.length + 1,
+      title: value,
+      completed: '',
```

Comment:
The 'completed' field in the 'newTodo' object should be a boolean, not a string. Consider setting the initial value to false.
![image](https://github.com/mate-academy/react_todo-app/assets/99403198/09cbc9e1-a44c-467b-a91c-f5ebfeb7a660)